packages/capnproto/build.sh

```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE=https://capnproto.org/
+TERMUX_PKG_DESCRIPTION="Cap'n Proto tool and library"
```

Comment:
Use normal description here. We all know that package contains cap'n proto stuff, but what is it?
```
Data interchange format and capability-based RPC system.
```